### DISCUSSION TITLE
Remove the 1st line using a Cue Flag- is it possible?

### DISCUSSION THREAD

**Discussion by Newbie112222** [cuelang/cue#966] [Q&A]
Hi All,
I have a use case where i use the cue export command to compare 2 files. My issue is that the final yaml file (after successfully running the export command) i generate has a field on the 1st line which i want to comment or remove from the file.
Any inputs would be appreciated:

Final File test.yaml:
config:   ( <------- remove this from the final file)
  url: google.com
  storage:
    type: postgres

**Comment by eonpatapon** [comment]
You can define `config` as a hidden field: `_config`

It then won't be exported.

**Reply by mpvl** [reply]
Alternatively, if `config` is still part of a public API, you can use `#config` as well.

**Comment by cueckoo** [comment]
This discussion has been migrated to https://github.com/cue-lang/cue/issues/966.

For more details about CUE's migration to a new home, please see https://github.com/cue-lang/cue/issues/1078.
